packages/datetime/test/dateRangePickerTests.tsx

```diff
@@ -904,74 +904,47 @@ describe("<DateRangePicker>", () => {
         });
 
         it("custom shortcuts set the displayed months correctly when start month changes", () => {
-            const dateRange = [
-                new Date(2016, Months.JANUARY, 1, 10, 20, 30),
-                new Date(2016, Months.DECEMBER, 31, 10, 20, 30),
-            ] as DateRange;
-
-            const test = (shouldChangeTime: boolean) => {
-                const { left, right } = render({
-                    initialMonth: new Date(2015, Months.JANUARY, 1),
-                    shortcuts: [{ label: "custom shortcut", dateRange, shouldChangeTime }],
-                }).clickShortcut();
-                assert.isTrue(onChangeSpy.calledOnce);
-                left.assertMonthYear(Months.JANUARY, 2016);
-                right.assertMonthYear(Months.FEBRUARY, 2016);
```

Comment:
time changes were never asserted here so this whole change was useless. reverted to pre #3223 state.

there is a test below that _does_ assert `includeTime` works as expected.